schedule/qam/15-SP4/qam-regression-other_fips.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: qam regression other tests with fips enabled for Vendor Affirmation
+schedule:
+- boot/boot_to_desktop
```

Comment:
openjdk is missing

Reply:
Yeah, for both (SLES and SLED)... x11 is not an issue in this case

Reply:
Let me do!

Reply:
http://openqa.suse.de/tests/10641681#